File: VB/E2311/Helper/MyNavBarSuperToolTipHelperItem.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraNavBar
Imports DevExpress.Utils
Imports System.Drawing.Design
Imports System.Collections

Namespace E2311
	Public Class MyNavBarSuperToolTipHelperItem
		Private _SuperTip As SuperToolTip
        <Localizable(True), Description("Gets or sets a SuperToolTip for the current control."), Editor("DevExpress.XtraEditors.Design.ToolTipContainerUITypeEditor, " + AssemblyInfo.SRAssemblyEditorsDesign, GetType(UITypeEditor))> _
  Public Property SuperTip() As SuperToolTip
            Get
                Return _SuperTip
            End Get
            Set(ByVal value As SuperToolTip)
                _SuperTip = value
            End Set
        End Property

		Private _Group As NavBarGroup
		Public Property Group() As NavBarGroup
			Get
				Return _Group
			End Get
			Set(ByVal value As NavBarGroup)
				_Group = value
			End Set
		End Property

		Public Sub New()

		End Sub
	End Class

End Namespace

File: CS/E2311/Helper/MyNavBarSuperToolTipHelperItem.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraNavBar;
using DevExpress.Utils;
using System.Drawing.Design;
using System.Collections;

namespace E2311
{
    public class MyNavBarSuperToolTipHelperItem
    {
        private SuperToolTip _SuperTip;
        [Localizable(true)]
        [Description("Gets or sets a SuperToolTip for the current control.")]
        [Editor("DevExpress.XtraEditors.Design.ToolTipContainerUITypeEditor, " + AssemblyInfo.SRAssemblyEditorsDesign, typeof(UITypeEditor))]
        public SuperToolTip SuperTip
        {
            get { return _SuperTip; }
            set { _SuperTip = value; }
        }

        private NavBarGroup _Group;
        public NavBarGroup Group
        {
            get { return _Group; }
            set { _Group = value; }
        }

        public MyNavBarSuperToolTipHelperItem()
        {

        }
    }

}
